frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -182,6 +182,9 @@ export const createOrUpdateBuildConfig = (
   let buildStrategyData;
 
   let desiredContextDir = contextDir;
+  const customBuildEnvs = imageEnv
+    ? Object.keys(imageEnv).map((k) => ({ name: k, value: imageEnv[k] }))
+    : [];
```

Comment:
```suggestion
  const customBuildEnvs = imageEnv
    ? Object.keys(imageEnv).map((k) => ({ name: k, value: imageEnv[k] }))
    : [];
```

As we noticed together in slack, this code should check if the value is not empty.

Otherwise a touch to the `NPM_RUN` field would broke the deployment. (Builld passes, but the app could not started with `NPM_RUN=` environment variable.

Reply:
Already solved :+1: